src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,24 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    completed,
+    title,
+  } = todo;
+
+  return (
+    <article
+      className={`TodoInfo ${classNames({ 'TodoInfo--completed': completed })}`}
```

Comment:
[Read more](https://www.npmjs.com/package/classnames) about correct useing classnames. Class `TodoInfo` also should be inside `classnames`